apps/nextjs-website/src/components/organisms/Auth/AuthStatus.tsx

```diff
@@ -0,0 +1,14 @@
+'use client';
```

Comment:
I would rather move this logic to `apps/nextjs-website/src/app/auth/login/page.tsx` because its just a side effect linked more on the behaviour of the sign in page

Reply:
I moved this logics to custom [hook](https://github.com/pagopa/developer-portal/blob/79f90b9faded9f7c3425f46818d1a88585c366ad/apps/nextjs-website/src/helpers/user.helper.ts#L71) in this [PR](https://github.com/pagopa/developer-portal/pull/706).

The best solution to handle this redirect would be a middleware.